ui/app/adapters/pki-ca-certificate.js

```diff
@@ -32,6 +32,11 @@ export default ApplicationAdapter.extend({
       data = { certificate: snapshot.attr('certificate') };
     } else {
       data = serializer.serialize(snapshot, requestType);
+
+      // The type parameter is serialized but is part of the URL. This means
+      // we'll get an unknown parameter warning back from the server if we
+      // send it. Remove it instead.
+      delete data['type'];
```

Comment:
😍  the comments! wondering if we should wrap the delete in a conditional, in case the form changes. Although the failure at attempting the delete a non-existent attr might be a good thing...
```suggestion
      if (data.type) delete data.type;
```

Reply:
The `type` field of the form is used above for URL building:

https://github.com/hashicorp/vault/blob/5f696cbad4a8e6172a90fe42ed6447ff08d44a86/ui/app/adapters/pki-ca-certificate.js#L7-L19 

Which is called from below:

https://github.com/hashicorp/vault/blob/5f696cbad4a8e6172a90fe42ed6447ff08d44a86/ui/app/adapters/pki-ca-certificate.js#L42

If the form changes, we've got bigger issues (URL will now be invalid, causing _everything_ to fail). My 2c? :-)

Reply:
Typically we'd do transformations like this in the serializer, but since that doesn't exist for this model type I think this is fine. Deleting an unknown key should not throw an error, but stylistically we should make it `delete data.type` instead. Thank you!

Reply:
```suggestion
      delete data.type;
```

Reply:
Thank you!